front/app/components/admin/Graphs/BarChart/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { isEmpty } from 'lodash-es';
+
+// components
+import {
+  ResponsiveContainer,
+  BarChart as _BarChart,
```

Comment:
Could this be named BarChartComponent or something like that? When we prefix with `_`, it usually means you are declaring something you won't use and you want eslint to ignore it